oauth2_http/java/com/google/auth/oauth2/ClientId.java

```diff
@@ -123,7 +131,7 @@ public static ClientId fromStream(InputStream stream) throws IOException {
    * @param clientId Text identifier of the Client ID.
    * @param clientSecret Secret to associated with the Client ID.
    */
-  public ClientId(String clientId, String clientSecret) {
+  protected ClientId(String clientId, String clientSecret) {
```

Comment:
Could you mark the constructors `@Deprecated` instead of deleting them / reducing visibility? It makes it easier to transition between versions if we deprecate first, then delete later.

Reply:
Done